src/client/components/Logo.tsx

```diff
@@ -0,0 +1,87 @@
+          fill="#fff"
+        />
+      </svg>
+    </div>
+  );
+};
+
+export const Logo = ({ isAnniversary }: { isAnniversary?: boolean }) => (
+  <Link
+    href="https://www.theguardian.com"
+    title="The Guardian Homepage"
+    subdued={true}
+    cssOverrides={css`
+      color: ${brandText.primary};
+      :hover {
+        color: ${brandText.primary};
```

Comment:
just wondering why have this hover selector if it doesn't change the colour

Reply:
This is there to override the hover value that you get when using `Link`

Reply:
ah yes, missed that :)